modules/arm_plugin/src/opset/pool_arm.cpp

```diff
@@ -12,9 +12,9 @@ opset::v1::ArmMaxPool::ArmMaxPool(const ov::Output<Node>& arg,
                                   const ov::Shape& pads_begin,
                                   const ov::Shape& pads_end,
                                   const ov::Shape& kernel,
+                                  const ov::PartialShape& output_shape,
```

Comment:
it's better to make it the last argument here

Reply:
all arguments after `output_shape` are optional. So if we want `output_shape` to not be optional, the last position to place it here is after `kernel` argument.